twake/frontend/src/app/scenes/Client/ChannelsBar/ChannelsBar.tsx

```diff
@@ -37,17 +40,30 @@ export default () => {
   }, []);
 
   return (
-    <Layout.Sider theme="light" width={220} className="channels_view">
-      <CurrentUser />
-      {!!companyId && !!workspaceId && (
-        <PerfectScrollbar component="div">
-          <ChannelsApps key={workspaceId} />
-          <Workspace key={'workspace_chans_' + workspaceId} />
-          <ChannelsUser key={companyId} />
-        </PerfectScrollbar>
-      )}
-      <Tutorial />
-      <Footer />
-    </Layout.Sider>
+    <ScrollWithHiddenComponents
```

Comment:
Here the scrollbar should not be for the whole bar but only the channels
Please keep the perfect scrollbar position and change the css of the go top go down buttons